org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/java/org/eclipse/winery/bpmn2bpel/BPMN4Tosca2BpelTest.java

```diff
@@ -13,50 +13,135 @@
  *******************************************************************************/
 package org.eclipse.winery.bpmn2bpel;
 
-import java.net.MalformedURLException;
+import org.eclipse.winery.bpmn2bpel.parser.ParseException;
+import org.eclipse.winery.bpmn2bpel.planwriter.PlanWriterException;
+
+
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.file.Paths;
+import java.io.File;
+import java.io.IOException;
 
-import org.eclipse.winery.bpmn2bpel.parser.ParseException;
-import org.eclipse.winery.bpmn2bpel.planwriter.PlanWriterException;
-
+import org.junit.rules.TemporaryFolder;
+import static org.junit.Assert.fail;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-
+import org.junit.Rule;
 
 public class BPMN4Tosca2BpelTest {
 
 	protected static String RESOURCES_DIR = "src/test/resources/bpmn4tosca";
 
+	/**
+	 *  Temporary Folder instance to keep out Zips from Sources.
+	 */
+	@Rule
+	private TemporaryFolder folder;
 
+	/**
+	 * Folder where Temporary Zip files will be created.
+	 */
+	private File temporaryFolder;
```

Comment:
Why this avariable and the variable "folder". I would assume that `folder` should be used instead of `temporaryFolder`.